docs/tasks/run-application/update-api-object-kubectl-patch.md

```diff
@@ -54,7 +54,7 @@ get terminated and replaced by new ones.
 At this point, each Pod has one Container that runs the nginx image. Now suppose
 you want each Pod to have two containers: one that runs nginx and one that runs redis.
 
-Create a file named `patch-file.yaml` that has this content:
+Create a file named `patch-file-containers.yaml` that has this content:
 
 ```shell
```

Comment:
Since you are here, how about change this to "```yaml"?

Reply:
Done